frontend-student/src/components/signup/Signup.jsx

```
@@ -27,149 +47,345 @@ export default function Signup() {
+                  onBlur={signupFormik.handleBlur}
+                  value={signupFormik.values.password}
+                />
+                <div className='absolute right-3 top-1/2 transform -translate-y-1/2 text-gray-500'>
+                  <svg
+                    xmlns='http://www.w3.org/2000/svg'
+                    className='h-5 w-5'
+                    fill='none'
+                    viewBox='0 0 24 24'
+                    stroke='currentColor'
+                  >
+                    <path
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)